ibm_vpd_utils.cpp

```diff
@@ -1023,5 +1023,21 @@ Binary getVpdDataInVector(const nlohmann::json& js, const std::string& file)
 
     return vpdVector;
 }
+void getNumOfEvenAndOddNumbrsInGivenNum(int& num, uint8_t& numOfEvenNumbrs,
```

Comment:
Same here, a balnk line.

Reply:
I was not pointing that, I was asking to have a blank line between between two functions, which you have fixed. Thanks.